docs/examples/resolutions.yml

```diff
@@ -0,0 +1,31 @@
+---
+# This file contains global resolutions for errors and rule violations occuring in the OSS scan.
+# These resolutions are applied to all scanned projects, so make sure to only
+# add resolutions that are not specific to a project. Project specific
+# resolutions can be configured in the .ort.yml file of the project.
+#
+# For detailed documentation, see:
+# https://github.com/heremaps/oss-review-toolkit/blob/master/docs/config-file-resolutions-yml.md
+
+# Structure of error resolution:
+#
+# errors:
+# - message: "A regular expression matching the error message."
+#   reason: "One of: BUILD_TOOL_ISSUE|CANT_FIX_ISSUE|SCANNER_ISSUE"
+#   comment: "A comment further explaining why the reason above is applicable."
+errors:
+- message: "^$"
+  reason: "CANT_FIX_ISSUE"
+  comment: "An example error resolution which resolves only errors with empty string error message."
+
+# Structure of policy rule violation resolution:
+#
+# rule_violations:
+# - message: "A regular expression matching the violation message."
+#   reason: "One of: CANT_FIX_EXCEPTION|DYNAMIC_LINKAGE_EXCEPTION|EXAMPLE_OF_EXCEPTION|LICENSE_ACQUIRED_EXCEPTION|PATENT_GRANT_EXCEPTION"
+#   comment: "A comment further explaining why the reason above is applicable."
+
+rule_violations:
+- message: "^$"
+  reason: "CANT_FIX_EXCEPTION"
+  comment: "An example policy violation resolution which resolves only errors with empty string rule message."
```

Comment:
"A policy violation resolution ... an empty error message."

It's confusing to use "rule violation" and "policy violation" for the same thing, we should stick to one term.

Reply:
Done consistently now using policy rule violation